removed  REG_SZ  added  REG_DWORD (#8533)
MicrosoftDocs/windows-itpro-docs: windows/privacy/manage-connections-from-windows-operating-system-components-to-microsoft-services.md

  -or-

> - Create a new REG_DWORD registry setting named **DisableEnhancedNotifications** in **HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\Windows Defender\Reporting** to a value of **1**.

Reviewer: The phrasing in this sentence seems to be made for a case where an existing key should be set to a specific value. When creating a new key, I am fairly certain we should phrase it like this:
```suggestion
- Create a new REG_DWORD registry setting named **DisableEnhancedNotifications** in **HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\Windows Defender\Reporting** and enter the decimal value **1**.
```
( I know, you are not the original author of this sentence, but we aim to improve the documentation whenever we can. )
